bullseye-slim/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.18.2-bullseye AS builder
+
+LABEL maintainer="Skynet Labs <devs@skynetlabs.com>"
+
+WORKDIR /root
+
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
+
+ARG branch
+
+RUN git clone https://gitlab.com/SkynetLabs/skyd.git --single-branch --depth 1 --branch ${branch} && \
+    make build --directory skyd
+
+FROM debian:bullseye-slim
+
+LABEL maintainer="Skynet Labs <devs@skynetlabs.com>"
+
+# copy mime types
+COPY --from=builder /etc/mime.types /etc/mime.types
```

Comment:
why do we need this?

Reply:
afaik skyd uses mime types table and afaik it does not get included in the binary but is accessed from host system from known locations

reference from docker-sia that says we need to install mailcap package for mime types support
https://github.com/NebulousLabs/docker-sia/blob/master/alpine/Dockerfile#L23-L26

Reply:
@ChrisSchinnerl reference https://go.dev/src/mime/type.go

```
// TypeByExtension returns the MIME type associated with the file extension ext.
// The extension ext should begin with a leading dot, as in ".html".
// When ext has no associated type, TypeByExtension returns "".
//
// Extensions are looked up first case-sensitively, then case-insensitively.
//
// The built-in table is small but on unix it is augmented by the local
// system's MIME-info database or mime.types file(s) if available under one or
// more of these names:
//
//   /usr/local/share/mime/globs2
//   /usr/share/mime/globs2
//   /etc/mime.types
//   /etc/apache2/mime.types
//   /etc/apache/mime.types
//
// On Windows, MIME types are extracted from the registry.
//
// Text types have the charset parameter set to "utf-8" by default.
```

Reply:
Interesting thanks